server/routes/linkTokens.js

```diff
@@ -24,9 +24,12 @@ router.post(
   '/',
   asyncWrapper(async (req, res) => {
     try {
-      const { userId, itemId } = req.body;
+      const { userId, itemId, isAuth, isIdentity } = req.body;
```

Comment:
looks like `isAuth` isn't getting used anywhere; is it supposed to do the similar conditional push to the products array like `identity`?

Reply:
oh, yeah, it is not being used here in the linkTokens route...we use "auth" regardless of whether we are using a processor or not.